ci/images/e2e-test-runner/run-e2e-test.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+SCRIPT_DIR="$(
+  cd "$(dirname "$0")" >/dev/null
+  pwd
+)"
+
+PROJECT_DIR="$(
+  cd "$SCRIPT_DIR/../../.." >/dev/null || exit 1
+  pwd
+)"
+
+# shellcheck source=ci/images/ci-runner/hack/bin/utils.sh
+source "$PROJECT_DIR/ci/images/ci-runner/hack/bin/utils.sh"
+
+export GITHUB_USER GITHUB_TOKEN QUAY_TOKEN QUAY_OAUTH_USER QUAY_OAUTH_TOKEN QUAY_OAUTH_TOKEN_RELEASE_SOURCE QUAY_OAUTH_TOKEN_RELEASE_DESTINATION
```

Comment:
Do we want to follow our existing shell script writing style For ex https://github.com/openshift-pipelines/pipeline-service/blob/main/.tekton/hack/setup_pipeline-service-ci.sh ?

Reply:
I need to take a look at that script